sympy/stats/rv.py

```diff
@@ -1428,6 +1429,55 @@ def __getattr__(self, attr):
             raise AttributeError("'%s' object has no attribute '%s'" % (
                 type(self).__name__, attr))
 
+
+class Distribution(Basic):
+
+    def sample(self, size=(), library='scipy'):
+        """ A random realization from the distribution """
+
+        module = import_module(library)
+        if library in {'scipy', 'numpy', 'pymc3'} and module is None:
+            raise ValueError("Failed to import %s" % library)
+
+        if library == 'scipy':
+            # scipy does not require map as it can handle using custom distributions.
+            # However, we will still use a map where we can.
+
+            # TODO: do this for drv.py and frv.py if necessary.
+            # TODO: add more distributions here if there are more
+            # See links below referring to sections beginning with "A common parametrization..."
+            # I will remove all these comments if everything is ok.
+
+            samps = self._do_sample_scipy(size)
```

Comment:
I think that `import scipy` can be written just above this line. This may help in avoid writing, `import scipy` for every implementation of `self._do_sample_scipy` as `scipy` will be imported into the namespace before go for sampling using it. Possibly, you may have tried it. Can you please let me know why it didn't work?

Reply:
Same goes for other libraries if it works.